app/assets/javascripts/components/common/wikidata_overview_stats.jsx

```diff
@@ -3,6 +3,45 @@ import PropTypes from 'prop-types';
 import OverviewStat from './OverviewStats/overview_stat';
 
 const WikidataOverviewStats = ({ statistics, isCourseOverview }) => {
+  // Helper function to check if all stats are zero
+  const allStatsAreZero = () => {
+    const statsToCheck = [
+      'total revisions',
+      'merged to',
+      'interwiki links added',
+      'items created',
+      'items cleared',
+      'claims created',
+      'claims changed',
+      'claims removed',
+      'labels added',
+      'labels changed',
+      'labels removed',
+      'descriptions added',
+      'descriptions changed',
+      'descriptions removed',
+      'aliases added',
+      'aliases changed',
+      'aliases removed',
+      'qualifiers added',
+      'references added',
+      'redirects created',
+      'reverts performed',
+      'restorations performed',
+      'other updates',
+      'lexeme items created'
+    ];
```

Comment:
This seems like it's duplicating a list of keys that might change over time, and then this list might not match.

Is there a good way to do this check without relying on an explicit list of keys? Can we just loop through all the keys that are present and check them for non-zeroes?

Reply:
These seems good. I will test it this week or next. Thanks!